testcases/kernel/syscalls/fallocate/fallocate01.c

```diff
@@ -142,15 +157,13 @@ void setup(void)
 {
 	/* Create temporary directories */
 	TEST_PAUSE;
+	// fallocate system call is failing temporary directory created using tst_tmpdir function.
```

Comment:
Change to
// fallocate system call is failing to create temporary directory under /tmp using tst_tmpdir function.
// so,  creating test directory in root filesystem as workaroud
mkdir(tempdir, 0777);